### ISSUE TITLE
Change remaining-time-display.js so that the time value announced by a screen reader is not negative

### ISSUE THREAD

**Issue by OwenEdwards** [videojs/video.js#5168]
## Description
As mentioned in #5157, the convention to display the remaining time in a video as a negative number is purely a visual convention; for screen reader users, where the remaining time has a hidden label, the actual amount of remaining time is positive, not negative.

So the `remaining-time-display.js` component should either add the negative sign in a way that is decorative and not announced by screen readers, or the visual (negative) number should be aria-hidden from screen readers and a hidden positive value should be added.

**Comment by smbea** [comment]
Hello, this is my approach for fixing the issue. I made it so the '-' is added on every time remaining update in a purely decorative way. I was wondering if this fits the what you had in mind for the solution or it should be a more efficient one.

You can see the screenshots of the solution working:

![screenshot 2018-12-05 at 21 12 19](https://user-images.githubusercontent.com/25825387/49544421-bb0fde00-f8d2-11e8-92bb-c2f5f277fe90.png)
![screenshot 2018-12-05 at 21 12 29](https://user-images.githubusercontent.com/25825387/49544422-bb0fde00-f8d2-11e8-8c04-e77d831f05e6.png)

And here are the changes made:

![screenshot 2018-12-05 at 21 13 54](https://user-images.githubusercontent.com/25825387/49544423-bb0fde00-f8d2-11e8-9451-290215665077.png)
![screenshot 2018-12-05 at 21 14 00](https://user-images.githubusercontent.com/25825387/49544424-bba87480-f8d2-11e8-8e10-d5b499be7601.png)
